src/Symfony/Component/BrowserKit/AbstractBrowser.php

```diff
@@ -445,7 +463,7 @@ protected function doRequestInProcess($request)
         }
 
         if (!$process->isSuccessful() || !preg_match('/^O\:\d+\:/', $process->getOutput())) {
-            throw new \RuntimeException(sprintf('OUTPUT: %s ERROR OUTPUT: %s.', $process->getOutput(), $process->getErrorOutput()));
+            throw new \RuntimeException(sprintf('OUTPUT: "%s" ERROR OUTPUT: "%s".', $process->getOutput(), $process->getErrorOutput()));
```

Comment:
I know, this is another change suggested by fabbot. I don't mind if we change the error message format, but I'm unsure if we really should.

Reply:
Did revert all fabbot changes.